homework/tests/Homework.spec.js

```diff
@@ -0,0 +1,145 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+const data = [
+  'Prototype',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9'
+]
+
+
+data.forEach(version => {
```

Comment:
You can add `index` here as a second variable, like:
`data.forEach((version, i) => {`
This way works well in our case since you written 10 test cases and we have 10 different versions of calculator.
Then, on every test, you can add that index inside the test case at the beginning so you won't need to use hardcoded test case number